packages/node-opentelemetry/README.md

```diff
@@ -150,6 +151,24 @@ app.use((req, res, next) => {
 
 ### (Optional) Advanced Instrumentation Configuration
 
+#### Adding Additional 3rd-Party Instrumentation Packages
+
+When manually instrumenting the SDK, use the ```additionalInstrumentations``` key to create an array of additional 3rd-part instrumentations. Check [here](https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-node#supported-instrumentations) to see the current auto instrumentation packages.
+
+```ts
+const { initSDK } = require('@hyperdx/node-opentelemetry');
+const { RemixInstrumentation } = require('opentelemetry-instrumentation-remix');
+
+
+initSDK({
+    consoleCapture: true, // optional, default: true
```

Comment:
this indentation looks like 4 spaces ? maybe make it 2 to be consistent with others